tests/unit/helpers.js

```diff
@@ -732,3 +732,13 @@ export async function matchingSagaAction(
   }
   return foundAction;
 }
+
+export const getFakeLogger = (params = {}) => {
```

Comment:
Good idea. I'm surprised we didn't have one. Could you file an issue to port all existing test code over to this? Maybe it would be a *good first bug*. I see about 8 calls that look like `fakeLog = { ... }`.

Reply:
Yep, this bothered me several times.